src/OrcasEngine/Resources/Constants.cs

```diff
@@ -87,10 +87,14 @@ internal static string AssemblyVersion
         {
             get
             {
-                Version version = new Version(ThisAssembly.AssemblyVersion);
+#if STANDALONEBUILD
```

Comment:
Is this right? `STANDALONEBUILD` seems wrong since it's just pulling from the current version number.